app/models/feedbook/employee.rb

```diff
@@ -1,10 +1,10 @@
 module Feedbook
   class Employee < ApplicationRecord
-    has_ancestry
-    
+
     has_many :employee_has_skills
     has_many :skills, through: :employee_has_skills
```

Comment:
`has_many through:` is intended to associate two models using a third, independent one, but `employee_has_skills` looks more like a pivot table that an actual model. I think using `has_and_belongs_to_many` will be more appropriate in this instance. Same for `position_has_employees`.

http://guides.rubyonrails.org/association_basics.html#choosing-between-has-many-through-and-has-and-belongs-to-many